pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -131,6 +123,34 @@ func (a *Actuator) Create(ctx context.Context, cluster *clusterv1.Cluster, machi
 	return nil
 }
 
+func (a *Actuator) getWorkerNodeToken(cluster *clusterv1.Cluster, controlPlaneURL string) (string, error) {
+	var bootstrapToken string
```

Comment:
Can this value be declared at the bottom? Doesn't seem necessary to be declared here, but I might be wrong.

Reply:
This was a copy pasta :)